lib/ifc-utils.ts

```
@@ -1504,13 +1512,12 @@ export function performAnalysis(
 
 // Export functions
-export function exportData(
+export async function exportData(
   elementsInput: IfcElement[] | { elements: IfcElement[] },
   format = "csv",
-  fileName = "export",
-  properties = "Name,Type,Material"
-): string | Promise<void> {
-  console.log("Exporting data:", format, fileName, properties);
+  fileName = "export"
+): Promise<string | ArrayBuffer | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)